addons/mass_mailing/tests/test_mailing_list.py

```diff
@@ -334,3 +335,21 @@ def test_mailing_update_optout(self):
             [('contact_id', '=', contact.id)]
         )
         self.assertEqual(subs.list_id, ml_1 + ml_2)
+
+
+@tagged('mailing_list')
+class TestContactAccess(MassMailCommon):
+
+    @users('user_marketing')
+    def test_mailing_contact_access(self):
+        # Check that mailing user can edit properties on mailing contact
+        value = [{'type': 'char', 'name': 'test', 'value': 'test', 'definition_changed': True}]
+        contact = self.env['mailing.contact'].create({'properties': copy.deepcopy(value)})
+        self.assertEqual(dict(contact.properties), {'test': 'test'})
```

Comment:
Could probably also check to update / remove properties so that we cover all rights :)